Makefile

```diff
@@ -0,0 +1,40 @@
+SHELL := /bin/bash
+
+CURRENT_PATH = $(shell pwd)
+REPO = # ghcr.io
+IMAGE = techneatium/techs-checks-latex
+TAG = latest
+FULL_IMAGE = ${REPO}${IMAGE}:${TAG}
+TARGET = F14_CheatSheet.tex
+
+.PHONY: build_container
+prepare_image: clean_fonts
+	docker build -t ${FULL_IMAGE} .
+
+.PHONY: build_pdf
+build_pdf:
+	mkdir -p ./build
+	source scripts/set_build_info.sh
+	docker run --rm -it \
+		-v ${CURRENT_PATH}:/source:rw,Z \
+		--entrypoint /bin/bash ${FULL_IMAGE} -c "\
+		cd /source &&\
+		scripts/set_build_info.sh &&\
+		cp -v /fonts/* /source/. &&\
+		lualatex --output-format=pdf --output-folder=/source/build ${TARGET} ;\
+		lualatex --output-format=pdf --output-folder=/source/build ${TARGET} ;\
+		lualatex --output-format=pdf --output-folder=/source/build ${TARGET}"
+
+.PHONY: build
+build: clean build_pdf
```

Comment:
Would it be possible to clean the fonts after building the pdf to reduce clutter? (This problem is also alleviated if the fonts are in a subfolder)